scarb/src/ops/package.rs

```diff
@@ -88,10 +96,47 @@ fn before_package(ws: &Workspace<'_>) -> Result<()> {
     Ok(())
 }
 
-#[tracing::instrument(level = "trace", skip(_opts, ws))]
+#[derive(Serialize)]
+struct VcsInfo {
+    sha1: String,
+    path_in_vcs: String,
+}
```

Comment:
Is there a reason to diverge from Cargo in regards of file format? 🤔 

I.e.
```
#[derive(Serialize)]
struct VcsInfo {
    git: GitVcsInfo,
    /// Path to the package within repo (empty string if root). / not \
    path_in_vcs: String,
}

#[derive(Serialize)]
struct GitVcsInfo {
    sha1: String,
}
```
Serializes to 
```
{
 "git": {
   "sha1": "aac20b6e7e543e6dd4118b246c77225e3a3a1302"
 },
 "path_in_vcs": ""
}
```
While we do
```
{
 "sha1": "aac20b6e7e543e6dd4118b246c77225e3a3a1302",
 "path_in_vcs": ""
}
```

Reply:
It seems like Cargo want to be prepared for supporting other VCS but we don't, that's why I've decided that this additional key doesn't really bring much onto the table.

Reply:
We don't support other VCS, but this doesn't mean we never will, so I'm siding with @maciektr.